src/screens/Dashboard/AssetsDashboard.tsx

```diff
@@ -89,19 +85,7 @@ const AssetsDashboard = observer(() => {
     }),
     orderColumnHelper.accessor("amount", {
       header: () => {
-        return (
-          allContractBalance.isGreaterThan(BN.ZERO) && (
-            <ButtonConfirm
-              disabled={isLoading}
-              style={{
-                minWidth: "92px",
-              }}
-              onClick={handleWithdrawAll}
-            >
-              {isLoading ? "Loading..." : "Withdraw All"}
-            </ButtonConfirm>
-          )
-        );
+        return <></>;
```

Comment:
Зачем колонка если тут ничего нет?

И лучше писать просто return
```suggestion
        return;
```